pyee/twisted.py

```diff
@@ -67,7 +67,7 @@ def _emit_run(
         except Exception:
             self.emit("failure", Failure())
         else:
-            if iscoroutine and iscoroutine(result):
+            if iscoroutine and iscoroutine(result):  # type: ignore
```

Comment:
mypy thinks `iscoroutine` will always return True. I don't know why it thinks this.

But also, if I drop the conditional import, I shouldn't need to check for `iscoroutine`.

Reply:
I think it was actually that `iscoroutine` would always be truthy. Getting rid of the conditional import may fix this.